addons/account/views/account_view.xml

```diff
@@ -223,6 +223,29 @@
                 </tree>
             </field>
         </record>
+        <record id="account_journal_view_kanban" model="ir.ui.view">
```

Comment:
@kic-odoo  Is it necessary to create almost the same view as account_bank_journal_view_kanban?

Reply:
@adr-odoo , both the menu are different and their fields are different and currently dashboard opens from both menus in config. so why don't we need two different kanban views? the fields displayed in kanban views are the fields which are currently displayed in list views.